frontend/src/app/articles/[id]/page.tsx

```
@@ -0,0 +1,43 @@
+  }
+
+  return (
+    <ArticleDetailPage
+      id={blog.link}
+      link={`/articles/${blog.id}`}
+      image={blog.image.url}
+      category={blog.category.name}
+      date={blog.date}
+      title={blog.title}
+    >
+      <div dangerouslySetInnerHTML={{ __html: blog.body }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)